commands/reminder.js

```diff
@@ -0,0 +1,115 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const reminders = require('../utils/reminders.js');
+const embed = require('../utils/embed.js');
+const ms = require('ms');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('reminder')
+		.setDescription('Create, view, and manage reminders.')
+		.addSubcommand(subcommand =>
+			subcommand.setName('create')
+				.setDescription('Create a reminder.')
+				.addStringOption(option =>
+					option.setName('reminder')
+						.setDescription('What should I remind you about?')
+						.setRequired(true)
+				)
+				.addStringOption(option =>
+					option.setName('time')
+						.setDescription('When should I remind you? (e.g. 5m, 1h, 2d)')
+						.setRequired(true)
+				)
+		)
+		.addSubcommand(subcommand =>
+			subcommand.setName('view')
+				.setDescription('View reminders.')
+		)
+		.addSubcommand(subcommand =>
+			subcommand.setName('clear')
+				.setDescription('Clear upcoming reminders.')
+				.addIntegerOption(option =>
+					option.setName('id')
+						.setDescription('The reminder you wish to clear\'s id.'))
+		),
+	async execute(interaction) {
+		await interaction.deferReply({
+			ephemeral: true
+		});
+
+		if (interaction.options.getSubcommand() === 'create') { // create reminder
```

Comment:
Could be cleaner to have a function for each sub command and a switch case Or look up table to prevent one mega function

Just a thought not saying you have to of course